### PR TITLE
Added tests to sentinel (FINALLY!)

### PR DESCRIPTION
**Pull request by Giulio2002**

### CODE REVIEW

File: cl/antiquary/tests/tests.go [hunk 1]
```diff
@@ -67,18 +69,21 @@ func (m *MockBlockReader) FrozenSlots() uint64 {
 	panic("implement me")
 }
 
-func LoadChain(blocks []*cltypes.SignedBeaconBlock, db kv.RwDB) *MockBlockReader {
+func LoadChain(blocks []*cltypes.SignedBeaconBlock, db kv.RwDB) (*MockBlockReader, afero.Fs) {
```
Comment (elee1766) [inline cl/antiquary/tests/tests.go:72]: if this is a testing utility function, should pass testing.T into it

File: cl/sentinel/gossip.go [hunk 2]
```diff
@@ -137,10 +137,55 @@ func (s *GossipManager) unsubscribe(topic string) {
...
 	delete(s.subscriptions, topic)
 }
 
+func (s *Sentinel) forkWatcher() {
+	prevDigest, err := fork.ComputeForkDigest(s.cfg.BeaconConfig, s.cfg.GenesisConfig)
+	if err != nil {
+		log.Error("[Gossip] Failed to calculate fork choice", "err", err)
+		return
+	}
+	fmt.Println("A")
```
Comment (elee1766) [inline cl/sentinel/gossip.go:160]: wha

**Review by elee1766 — APPROVED** [review]

File: cl/clparams/config.go [hunk 3]
```diff
@@ -539,6 +539,7 @@ func toBytes4(in []byte) (ret [4]byte) {
 func configForkSchedule(b *BeaconChainConfig) map[libcommon.Bytes4]uint64 {
 	fvs := map[libcommon.Bytes4]uint64{}
 	fvs[utils.Uint32ToBytes4(b.GenesisForkVersion)] = 0
+	fmt.Println(b.AltairForkEpoch)
```
Comment (bayram98) [inline cl/clparams/config.go:542]: Maybe we could remove this.

**Review by bayram98 — APPROVED** [review]
Lgtm!

File: cl/sentinel/gossip.go (hunk 2 above)
Comment (Giulio2002) [reply cl/sentinel/gossip.go:160]: A

File: cl/clparams/config.go (hunk 3 above)
Comment (Giulio2002) [reply cl/clparams/config.go:542]: good catch

**Pull request merged**